### ISSUE TITLE
Running make command and nothing happens

### ISSUE THREAD

**Issue by Ryefendi** [kgretzky/evilginx2#1004]
When I run the Make command. It takes a second then nothing happens. I’m in the evilginx2 directory, Golang up to date. I’m not sure what’s wrong
![camphoto_1385675177](https://github.com/kgretzky/evilginx2/assets/157671540/79a42739-6ad7-481d-acd5-3d9a5d05a622)

**Comment by casiar** [comment]
It's done. Check your /build
then run with ./evilginx

**Comment by kpomeroy1979** [comment]
I don't think you get any output after you run 'make'. Check the ./bin/ directory to see if the evilginx binary is there. This is mentioned in the documentation. 

**# cd /home/lappy/evilginx2/bin/
# ./evilginx2**

**Comment by Ethansi2947E** [comment]
@kpomeroy1979 very true

**Comment by ttinkywinky45** [comment]
**Hi 👽 ! If you are interested in ⚙️Advanced Phishing⚙️ and the ✨Evilginx3✨ tool then I offer you Google Gmail and Custom Phishlets with a ✅Free Tutorial✅ which arrives if the 📈Telegram reaches 50 Subscribers📈
⏳ GOOGLE GMAIL PHISHLET 2024 WITH PROOF ⏳
 ->
⛔https://www.youtube.com/watch?v=nA47UknIE6A⛔
---
🔓 Ask your questions and a free tutorial from 50 subs on telegram !!!
->
📩https://t.me/Noremorse1011📩**

![evilginx_background](https://github.com/kgretzky/evilginx2/assets/144027798/0e448f68-ecdc-49a4-a4f4-23dbed7cb971)
